### PR TITLE
[#6611] Removed potential for segfault with l1 descriptors (4-2-stable)

### PR DESCRIPTION
**Pull request by korydraughn**
TODO:
- [x] Determine if `copy_l1desc()` should be included in this PR
- [x] Run tests

### CODE REVIEW

**Comment by korydraughn** [comment]
`copy_l1desc()` is not needed for 4-2-stable.

**Comment by korydraughn** [comment]
All core tests and unit tests passed.

Adding #

File: server/core/include/objDesc.hpp [hunk 1]
```diff
@@ -28,8 +23,12 @@
 #define REG_CHKSUM      1
 #define VERIFY_CHKSUM   2
 
-/* values in l1desc_t is the desired value. values in dataObjInfo are
- * the values in rcat */
+// Values in l1desc_t is the desired value.
```
Comment (korydraughn) [inline server/core/include/objDesc.hpp:26]: Will tweak a little.

Comment (korydraughn) [reply server/core/include/objDesc.hpp]: Done

**Comment by korydraughn** [comment]
Any last thoughts before I add the pounds?

**Review by trel — DISMISSED** [review]

**Comment by korydraughn** [comment]
Added #'s

**Review by alanking — APPROVED** [review]

**Pull request merged**